.github/workflows/ci.yml

```diff
@@ -0,0 +1,92 @@
+---
+name: OpenWISP Firmware Upgrader CI Build
+
+on:
+  push:
+    branches:
+      - master
+      - dev
+  pull_request:
+    branches:
+      - master
+      - dev
+
+jobs:
+  build:
+    name: Python==${{ matrix.python-version }} | ${{ matrix.django-version }}
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+        django-version:
+          - django~=3.0
+          - django~=3.1
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install system packages
+        run: |
+          sudo apt-get update && sudo apt-get upgrade
+          sudo apt-get -qq install \
+          sqlite3 \
+          gdal-bin \
+          libproj-dev \
+          libgeos-dev \
+          libspatialite-dev \
+          spatialite-bin \
+          libsqlite3-mod-spatialite
+
+      - name: Install python dependencies
+        run: |
+          pip install -U pip wheel setuptools
+          pip install ${{ matrix.django-version }}
+          pip install -U -r requirements-test.txt
+
+      - name: Install npm dependencies
+        run: sudo npm install -g jslint
```

Comment:
Are we actually running jslint? I might be a little out of the loop :smile:

Reply:
thanks @Purhan, no we're not: https://github.com/openwisp/openwisp-firmware-upgrader/blob/master/run-qa-checks
there's no JS in this repo at the moment, only some CSS (would be good to run the css linter)

Reply:
Then it is safe to remove `.jslintrc` file too.